machinelearning/caffe/README.md

```diff
@@ -0,0 +1,228 @@
+# Caffe
+
+Caffe is a deep learning framework that enables the possibility to run models in GPU/CPU, it enhances the modularity and speed of models.
+
+#### What you need:
+
+To build this container, you will need the following:
+
+ - Singularity, which you can download and install from [here](https://github.com/sylabs/singularity).
+ - A text editor, like: `micro`, `vim` or `nano`.
+ - Root access.
+
+#### Set up:
+
+Create a folder where all of your files to the caffe singularity container including the def file will reside. In this example, we will call the folder `caffehost` and inside we will create two (2) additional folders to run the example: `data` and `examples`.
+
+Your `caffehost` folder should look like this:
+
+```
+caffehost/
```

Comment:
Start this as `/caffehost`. You're using paths like `/caffehost` further on, but here you have a relative path.